rps.js

```diff
@@ -0,0 +1,98 @@
+/*Method 1
+```
+Name: lString Length 
+Description: returns the length of a string
+Example using it:
+```
+let nameLength = 'huda alismail';
+console.log(nameLength.length);
+13
+
+
+Method 2
+```
+Name: Replacing String
+Description: replaces a specified value with another value in a string
+Example using it:
+```
+let xx = 'I love green';
+console.log('replace green with blue')
+console.log(xx.replace('green','blue'));
+
+Method 3
+```
+Name:toUpperCase
+Description: convert string to uppercase
+Example using it:
+```
+let name = 'huda';
+console.log(name.toUpperCase());
+*/
+
+//Dice Roller
+
+const rollDice=function (Roll){
+    let min=1;
+    let max=6;
+    let number=[];
+
+
+   if(Roll>1){
+
+    for(i=1;i<=Roll;i++){
+
+    let num1=Math.random() * (max- min) + min;
+    number.push(num1);
+
+}
+let sum=number.reduce((a, b) => a + b, 0);
+console.log('Dice rolled are '+number+'. Sum is '+sum);
+    };
+};
+
+
+//rock, paper, or scissors 
+
+const playGame =function (user_choice){ 
```

Comment:
your current code only plays scissors and lets the player always win.

there is two ways you can fix this 

option 1:
add a break after every switch statement 
```
   switch (Math.floor(Math.random() * 3)) {
        case 0:
            computer_choice= 'rock';
            break
        case 1:
            computer_choice= 'paper';
            break
        case 2:
            computer_choice= 'scissors';
            break
        }
```

or option 2:
make the random function in the computer_choice, and remove the switch statement
```
let computer_choice=choices[Math.floor(Math.random() * 3)]
```

either one will work